backend/README.md

```diff
@@ -1,137 +1,152 @@
 # Backend
 
+## Installation
+{% tabs %}
+{% tab title="Docker" %}
 
-## Installation with Docker
+Run the following command to install everything through docker.
 
-Make sure you are on a [node](https://nodejs.org/en/) version &gt;= `v10.12.0`:
-
-```text
-  node --version
-```
-
-Run:
+The installation takes a bit longer on the first pass or on rebuild ...
 
 ```bash
-docker-compose up
-
-# create indices etc.
-docker-compose exec neo4j migrate
+$ docker-compose up
 
-# if you want seed data
-# open another terminal and run
-docker-compose exec backend yarn run db:seed
+# rebuild the containers for a cleanup
+$ docker-compose up --build
 ```
-
-App is [running on port 4000](http://localhost:4000/)
-
-To wipe out your neo4j database run:
+Open another terminal and create unique indices with:
 
 ```bash
-docker-compose down -v
+$ docker-compose exec neo4j migrate
 ```
 
-## Installation without Docker
+{% endtab %}
+
+{% tab title="Without Docker" %}
 
-Install dependencies:
+For the local installation you need a recent version of [node](https://nodejs.org/en/)
+(&gt;= `v10.12.0`) and [Neo4J](https://neo4j.com/) along with
+[Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures) plugin installed
+on your system.
 
 Download [Neo4j Community Edition](https://neo4j.com/download-center/#releases) and unpack the files.
 
 Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extracted Neo4j-Server
+Note that grand-stack-starter does not currently bundle a distribution of Neo4j. You can download [Neo4j Desktop](https://neo4j.com/download/) and run locally for development, spin up a [hosted Neo4j Sandbox instance](https://neo4j.com/download/), run Neo4j in one of the [many cloud options](https://neo4j.com/developer/guide-cloud-deployment/), [spin up Neo4j in a Docker container](https://neo4j.com/developer/docker/) or on Debian-based systems install [Neo4j from the Debian Repository](http://debian.neo4j.org/). Just be sure to update the Neo4j connection string and credentials accordingly in `.env`.
+Start Neo4J and confirm the database is running at [http://localhost:7474](http://localhost:7474).
 
-Start Neo4j
-
-```text
-neo4j\bin\neo4j start
-```
-
-and confirm it's running [here](http://localhost:7474)
-
+Now install node dependencies with [yarn](https://yarnpkg.com/en/):
 ```bash
-yarn install
-# -or-
-npm install
+$ cd backend
+$ yarn install
 ```
 
-Copy:
-
-```text
-cp .env.template .env
+Copy Environment Variables:
+```bash
+# in backend/
+$ cp .env.template .env
 ```
 
-Configure the file `.env` according to your needs and your local setup.
+Configure the new files according to your needs and your local setup.
 
-Start the GraphQL service:
+Create unique indices with:
 
 ```bash
-yarn dev
-# -or-
-npm dev
+$ ./neo4j/migrate.sh
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-04-15T23:51:36Z" title="Tuesday, April 16th 2019, 1:51:36 am +02:00">Apr 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,137 +1,155 @@
 # Backend
 
+## Installation
+{% tabs %}
+{% tab title="Docker" %}
 
-## Installation with Docker
+Run the following command to install everything through docker.
 
-Make sure you are on a [node](https://nodejs.org/en/) version &gt;= `v10.12.0`:
-
-```text
-  node --version
-```
-
-Run:
+The installation takes a bit longer on the first pass or on rebuild ...
 
 ```bash
-docker-compose up
-
-# create indices etc.
-docker-compose exec neo4j migrate
+$ docker-compose up
 
-# if you want seed data
-# open another terminal and run
-docker-compose exec backend yarn run db:seed
+# rebuild the containers for a cleanup
+$ docker-compose up --build
 ```
-
-App is [running on port 4000](http://localhost:4000/)
-
-To wipe out your neo4j database run:
+Open another terminal and create unique indices with:
 
 ```bash
-docker-compose down -v
+$ docker-compose exec neo4j migrate
 ```
 
-## Installation without Docker
+{% endtab %}
+
+{% tab title="Without Docker" %}
 
-Install dependencies:
+For the local installation you need a recent version of [node](https://nodejs.org/en/)
+(&gt;= `v10.12.0`) and [Neo4J](https://neo4j.com/) along with
+[Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures) plugin installed
+on your system.
 
 Download [Neo4j Community Edition](https://neo4j.com/download-center/#releases) and unpack the files.
 
 Download [Neo4j Apoc](https://github.com/neo4j-contrib/neo4j-apoc-procedures/releases) and drop the file into the `plugins` folder of the just extracted Neo4j-Server
+Note that grand-stack-starter does not currently bundle a distribution of Neo4j. You can download [Neo4j Desktop](https://neo4j.com/download/) and run locally for development, spin up a [hosted Neo4j Sandbox instance](https://neo4j.com/download/), run Neo4j in one of the [many cloud options](https://neo4j.com/developer/guide-cloud-deployment/), [spin up Neo4j in a Docker container](https://neo4j.com/developer/docker/) or on Debian-based systems install [Neo4j from the Debian Repository](http://debian.neo4j.org/). Just be sure to update the Neo4j connection string and credentials accordingly in `.env`.
+Start Neo4J and confirm the database is running at [http://localhost:7474](http://localhost:7474).
 
-Start Neo4j
-
-```text
-neo4j\bin\neo4j start
+Now install node dependencies with [yarn](https://yarnpkg.com/en/):
+```bash
+# in 'Human-Connection/'
+$ yarn install
+$ cd backend && yarn install
+$ cd ../webapp && yarn install
+$ cd ..
 ```
 
-and confirm it's running [here](http://localhost:7474)
-
+Copy Environment Variables:
 ```bash
-yarn install
-# -or-
-npm install
+$ cp cypress.env.template.json cypress.env.json
+$ cp backend/.env.template backend/.env
+$ cp webapp/.env.template webapp/.env
 ```
+Configure the new files according to your needs and your local setup.
 
-Copy:
+Create unique indices with:
 
-```text
-cp .env.template .env
+```bash
+$ ./neo4j/migrate.sh
```


some commands are assuming a person is at the root level and some from the `backend` directory...
Also, most of this is a replica of `installation.md` at the root level...
what is the purpose of having the duplication @roschaefer

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-16T00:16:55Z" title="Tuesday, April 16th 2019, 2:16:55 am +02:00">Apr 16, 2019</time>_

---


This PR tries to remove duplication. See [here](https://docs.human-connection.org/human-connection/v/docs-improve_installation_instructions/installation) it's not in the `installation.md` anymore.

The reason for having this in `./backend` is to have it as close to the code as possible.

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-04-16T00:18:14Z" title="Tuesday, April 16th 2019, 2:18:14 am +02:00">Apr 16, 2019</time>_

---


But the folder must be fixed. It should be a comment in the code block
```bash
# in path/to/some/where
....
````